src/mixins/enum-mixin.ts

```diff
@@ -46,6 +46,19 @@ export default class EnumMixin extends Vue {
     return item.status === FilingStatus.PENDING_CORRECTION
   }
 
+  /**
+   * Converts the entity type to an entity title.
+   * @param type the entity type to convert
+   */
+  entityTypeToTitle (type: EntityTypes): string {
+    switch (type) {
+      case EntityTypes.COOP: return 'Numbered Cooperative'
+      case EntityTypes.BCOMP: return 'Numbered Benefit Company'
+      case EntityTypes.CORP: return 'Numbered Corporation'
+    }
+    return 'Unknown' // should never happen
```

Comment:
Would it make sense to use a `default: return 'Unknown' in the event none of the cases are matched?

Reply:
I think both ways are equivalent.

Interestingly, the latest VSCode extensions show that the 'return' is unreachable code. I believe this is because it knows the values for 'type' and that they're all handled. Here's a screenshot of doing it both ways:

![image](https://user-images.githubusercontent.com/10002889/85601481-47900780-b603-11ea-828c-ee45cfe4019c.png)

Reply:
Fair enough, i'm good with either or :) Resolved!